File: python/BubbleSort.py
def bubbleSort(arr): 
    n = len(arr) 
  
    for i in range(n): 
  
        for j in range(0, n-i-1): 
   
            if arr[j] > arr[j+1] : 
                arr[j], arr[j+1] = arr[j+1], arr[j] 
 
arr = [14,1,13,7,50,12,60] 
  
bubbleSort(arr) 
  
print ("Sorted array is:") 
for i in range(len(arr)): 
    print ("%d" %arr[i]),  

File: java/BubbleSort.java
public class BubbleSort 
{   
    public static void main(String args[]) 
    { 
        int arr[] = {60,30,2,7,12,11,0,20}; 
        int n = arr.length;
        for (int i = 0; i < n-1; i++) 
            for (int j = 0; j < n-i-1; j++) 
                if (arr[j] > arr[j+1]){
                    // swaping by taking a temp variable
                    int temp = arr[j]; 
                    arr[j] = arr[j+1]; 
                    arr[j+1] = temp; 
                } 
        System.out.println("Sorted array is:"); 
        for (int i=0; i<arr.length; ++i) 
            System.out.println(arr[i]);  
    } 
} 